doc/release-notes/0.11/0.11.md

```diff
@@ -0,0 +1,164 @@
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.11.0 release notes
+
+These release notes support the [Eclipse OpenJ9 0.11.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.11.0/plan).
+
+
+## Binaries and supported environments
+
+OpenJ9 release 0.11.0 supports OpenJDK 8 and OpenJDK 11. Binaries are available at the AdoptOpenJDK project:
+
+- [OpenJDK with OpenJ9 version 8](https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=openj9)
+- [OpenJDK with OpenJ9 version 11](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=openj9)
+
+Builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.11.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.11/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/3118">#3118</a></td>
+<td valign="top">OpenSSL V1.1.1 support</td>
+<td valign="top">OpenJDK8 (All platforms)</td>
+<td valign="top">New command line options are available to enable OpenSSL support for improved cryptographic performance.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1646">#1646</a></td>
+<td valign="top">The shared classes cache is now enabled by default for bootstrap classes</td>
```

Comment:
Please remove this note as I have/am reverting shared classes enabled by default since it appears to have caused a number of failures and crashes.

Reply:
@SueChaplain I think you mentioned having removed this, did you forget to push?